README.md

```diff
@@ -29,21 +121,23 @@ As the maintainer of this project, please make a few updates:
 - `test_data`/`test_size` is not supported at the same time as `cv_split_column_names`
 - `test_data`/`test_size` is not supported at the same time as `cv_splits_indices`
 - Test runs are not supported for AutoML runs which have streaming enabled.
-
+- Model testing is not supported for image classification tasks.
+- `enable_dnn=True` is not compatible with the model testing feature at this time.
 
 ## Contributing
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+We welcome contributions and suggestions! See `CONTRIBUTING.md` for more details.
+
+### Issues and feedback
+
+All forms of feedback are welcome through this repo's issues:
+https://github.com/Azure/automl-devplat2-preview/issues
```

Comment:
This needs to be ISSUES from https://github.com/Azure/automl-testdataset-preview, not DevPlat2.0 PREVIEW site.